### ISSUE TITLE
Can't login from JciHitachi 

### ISSUE THREAD

**Issue by JerryC1225** [qqaatw/JciHitachiHA#91]
### 🐛 Describe the bug

The app has not been working since yesterday, Oct 29. I have followed the troubleshooting steps, but the log shows a “login fail” error.

Here are the steps I have tried:

	1.	Restarted the Home Assistant.
	2.	Re-downloaded the JicHitachi app.
	3.	Removed and reinstalled the app, then reconfigured it.

When I attempt step 3, I receive a “Login Failed: Incorrect email or password” message, even though I have confirmed my login information is 100% correct. I am able to log in successfully on the official app, “雲端智慧控.”

Could you please help resolve this issue? Thank you.

### Environment Information

HACS 1.34.0
JciHitachiHA 1.9.0

**Comment by qqaatw** [comment]
Hi can you check the log? The system status check is indicating that the cloud worked properly.

**Comment by tonyjrchen** [comment]
It seems I have same issue "cannot login in"
below are the logs after restarting HA.

========================
Logger: homeassistant.setup
Source: setup.py:269
First occurred: 4:08:17 PM (1 occurrences)
Last logged: 4:08:17 PM

Setup failed for custom integration 'jcihitachi_tw': Integration failed to initialize.

===========================
This error originated from a custom integration.

Logger: custom_components.jcihitachi_tw
Source: custom_components/jcihitachi_tw/__init__.py:99
integration: JciHitachi TW (documentation, issues)
First occurred: 4:08:17 PM (1 occurrences)
Last logged: 4:08:17 PM

Failed to login API: An event occurred but wasn't accompanied with data when refreshing Study support code.

==============================

Logger: homeassistant.setup
Source: runner.py:189
First occurred: 4:07:57 PM (1 occurrences)
Last logged: 4:07:57 PM

Setup of jcihitachi_tw is taking over 10 seconds.

**Comment by hikevin117** [comment]
the same problem.

記錄器: custom_components.jcihitachi_tw
來源: helpers/update_coordinator.py:386
整合: JciHitachi TW (相關文件, 問題)
首次發生: 2024年10月27日 下午9:50:42 (33 事件)
最後紀錄: 2024年11月2日 下午8:40:52

Error fetching jcihitachi_tw data: Command executed timed out when regularly fetching data.
Error fetching jcihitachi_tw data: Error communicating with API: An error occurred when reauthenticating with AWS Cognito Service: TooManyRequestsException Too many requests

**Comment by SugiuraAyano** [comment]
I have this issue too. 
If I downgrade the ha core version to 2024.10.4 the integration is working.

**Comment by motocyman** [comment]
I got the same issue.

Originally It worked on core 2024.11.3
But it suddenly failed due to "login failed".
upgrade to core 2024.12.3 => failed too.
downgrade back to 2024.11.3 => failed too.
downgrade back to 2024.10.4 => works fine. (Thanks @SugiuraAyano )

**Comment by SugiuraAyano** [comment]
After update 2 weeks ago this seems solved.

**Comment by motocyman** [comment]
Now it works in 2025.1.3 + jciHitachi v1.9.0 + OS 14.2.

My tests are as follows.
upgrade to core 2024.12.3 => failed too.
downgrade back to 2024.11.3 => failed too.
downgrade back to 2024.10.4 => works fine.

core 2025.1.4 + jciHitachi v1.9.0 + OS 14.1 => cannot work in start ONVIF, core's issue
core 2025.1.4 + jciHitachi master branch + OS 14.1 => cannot work in start ONVIF, core's issue
core 2025.2.1 + jciHitachi v1.9.0 + OS 14.1 => core works but jciHitachi cannot work. If force to add jciHitachi, after entering account and password, it shows "immediately" proxies issue.
core 2025.2.1 + jciHitachi v1.9.0 + OS 14.1 + force libjciHitachi 1.7.0 => the same as above core 2.1 case.
core 2025.1.3 + jciHitachi v1.9.0 + OS 14.1 => after entering account, wait for a long time, it shows unknown error occur. I guess "async_setup_entry" problem.
core 2025.1.3 + jciHitachi v1.9.0 + **OS 14.2** => it works fine. Seems related to OS.

**Comment by phokui** [comment]
I'm new to Home Assistant and it failed in 2025.2.4 + OS 14.2.

**Comment by SugiuraAyano** [comment]
Hi @phokui 
Please check [this issue](https://github.com/qqaatw/JciHitachiHA/issues/97) for further details.
TLDR
Change the libjcihitachi to 1.7.0 and install this version manually.

**Comment by qqaatw** [comment]
Please update to the latest version.
